eox_core/social_tpa_backends.py

```diff
@@ -9,7 +9,9 @@
 
 try:
     from social_core.backends.open_id_connect import OpenIdConnectAuth
+    from social_core.exceptions import AuthMissingParameter
```

Comment:
Do you think we need to split this into two try blocks, if the auth missing fails but OpenId works then we would override it, but that is very unlikely to happen, right?

Reply:
It's very unlikely, but covering that 1/1000 case doesn't require too much effort.